pkg/genericapiserver/genericapiserver.go

```diff
@@ -385,6 +354,11 @@ func (s *GenericAPIServer) Run(options *options.ServerRunOptions) {
 	<-insecureStartedCh
 	s.RunPostStartHooks(PostStartHookContext{})
 
+	// err == systemd.SdNotifyNoSocket when not running on a systemd system
+	if err := systemd.SdNotify("READY=1\n"); err != nil && err != systemd.SdNotifyNoSocket {
```

Comment:
This changes the "I'm up" semantic  Previously, we'd send it before we'd even gotten the http server up.  Why make this change?

@sdodson Do we rely on systemd saying its ready before we attempt to start the api server for some reason?